app/src/main-process/menu/build-default-menu.ts

```diff
@@ -334,6 +335,15 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+Shift+M',
         click: emit('merge-branch'),
       },
+      {
+        label: __DARWIN__
+          ? 'Rebase Current Branch…'
+          : 'R&ebase current branch…',
+        id: 'rebase-branch',
+        accelerator: 'CmdOrCtrl+Shift+E',
```

Comment:
I settled on `E` here because `R` or `B` are already assigned to other shortcuts